tests/test_solver.py

```diff
@@ -466,6 +466,23 @@ def test_unsat_core():
         yield raw_unsat_core, s
 
 
+def test_issue49():
```

Comment:
Can you please rename it to a more meaningful name? I agree "issue49" is clear, but I prefer a short string that indicates what the bug is :-)